docs/questions/query-builder/expressions/countif.md

```diff
@@ -0,0 +1,273 @@
+| 2   | Basic       |
+| 3   | Basic       |
+| 4   | Business    |
+| 5   | Premium     |
+
+> [Aggregations](../expressions-list.md#aggregations) like `CountIf` should be added to the query builder's [**Summarize** menu](../../query-builder/introduction.md#summarizing-and-grouping-by) > **Custom Expression** (scroll down in the menu if needed).
+
+## Parameters
+
+`condition` is a [function](../expressions-list.md#functions) or [conditional statement](../expressions.md#conditional-operators) that returns a boolean value (`true` or `false`).
+
+## Multiple conditions
+
+| ID  | Plan        | Active Subscription |
+|-----|-------------| --------------------|
+| 1   | Basic       | true                |
+| 2   | Basic       | true                | 
+| 3   | Basic       | false               |
+| 4   | Business    | false               |
+| 5   | Premium     | true                |
+
+### Mandatory matches
+
+To count the total rows in a table that match multiple mandatory conditions, combine the conditions using the `AND` operator:
+
+```
+CountIf(([Plan] = "Basic" AND [Active Subscription] = true))
+```
+
+This expression will return 2 on the sample data above (the total number of Basic plans that have an active subscription).
+
+### Optional matches
+
+To count the total rows in a table that match multiple optional conditions, combine the conditions using the `OR` operator:
+
+```
+CountIf(([Plan] = "Basic" OR [Active Subscription] = true))
+```
+
+Returns 4 on the sample data.
+
+### Some mandatory and some optional matches
+
+To combine mandatory and optional conditions, group the conditions using parentheses:
+
+```
+CountIf(([Plan] = "Basic" OR [Plan] = "Business") AND [Active Subscription] = "false"))
+```
+
+Returns 2 on the sample data.
+
+> Tip: make it a habit to put parentheses around your `AND` and `OR` groups to avoid making mandatory conditions optional (or vice versa).
+
+## Conditional count by group
+
+In general, to get a conditional count for a category or group, such as the number of inactive subscriptions per plan, you'll:
+
+1. Write a `CountIf` expression with your conditions.
+2. Add a [**Group by**](../../query-builder/introduction.md#summarizing-and-grouping-by) column in the query builder.
+
+Using the sample data:
+
+| ID  | Plan        | Active Subscription |
+|-----|-------------| --------------------|
+| 1   | Basic       | true                |
+| 2   | Basic       | true                | 
+| 3   | Basic       | false               |
+| 4   | Business    | false               |
+| 5   | Premium     | true                |
+
+Count the total number of inactive subscriptions per plan:
+
+```
+CountIf([Active Subscription] = false)
+```
+
+Alternatively, if your **Active Subscription** column contains `null` (empty) values that represent inactive plans, you could use:
+
+```
+{% raw %}CountIf([Payment], [Plan] != true){% endraw %}
+```
+
+> The "not equal" operator `!=` should be written as "!=".
```

Comment:
```suggestion
> The "not equal" operator should be written as `!=`.
```

Reply:
Oh yeah! This bit is weird because I have to explain why the code sample uses ≠ 

The actual formula should use != (which is what you get if you copy/paste the code sample)

The raw tags were a vestigial effort to force the != to show up 😅 

<img width="684" alt="Screenshot 2023-03-10 at 5 06 19 PM" src="https://user-images.githubusercontent.com/22361741/224437155-4330c80c-2435-4d2f-8f86-8750261d200a.png">